Cargo.toml

```diff
@@ -22,6 +22,7 @@ repl = ["dep:crossterm", "dep:ctrlc", "dep:rustyline"]
 hostname = ["dep:hostname"]
 tls = ["dep:native-tls"]
 http = ["dep:warp", "dep:reqwest"]
+num = []
```

Comment:
I am surprised to see this feature being added again without any further code changes adding use of this feature.

This feature was removed in c41aba6b90d2b29f893394d34a80c774b4506d7a and I was under the impression that I had removed the last left behind usages of it in a93585080c7fbf01d62c8e93569aeaa439eef538

Reply:
Oh, I saw a warning during compilation and that's why added, I will re-check if it is still relevant on a current `master` branch, because this code was rebased from some earlier commits.

Reply:
I've reverted it. Indeed it was a warning in the past, but now I don't see it anymore.